Jenkinsfile

```diff
@@ -124,175 +124,178 @@ lock(resource: "Lisk-Core-Nodes", inversePrecedence: true) {
 		)
 	}
 
```

Comment:
Jenkyns file bad synced. Needed to check with #25 . Some tests are not being tested.